huaweicloud/services/cc/resource_huaweicloud_cc_authorization.go

```diff
@@ -0,0 +1,296 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product CC
+// ---------------------------------------------------------------
+
+package cc
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceAuthorization() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAuthorizationCreate,
+		UpdateContext: resourceAuthorizationUpdate,
+		ReadContext:   resourceAuthorizationRead,
+		DeleteContext: resourceAuthorizationDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `The name of the cross-account authorization.`,
+			},
+			"instance_type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The instance type.`,
+			},
+			"instance_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The instance ID.`,
+			},
+			"project_id": {
```

Comment:
can `region_id` and `project_id` be get from config or client?

Reply:
done